src/pages/UserMyPets.jsx

```diff
@@ -37,23 +39,34 @@ export const UserMyPets = () => {
       <Typography paragraph marginLeft="20px" marginTop="20px" variant="h4" color="#16bac6">
         My Pets
       </Typography>
-      <Box width='70vw'>
-      <Grid container spacing={4} paddingLeft="40px" paddingRight="40px" gridAutoColumns="2" margin="0">
-        {pets.length > 0 ? (
-          pets.map((pet) => (
-            <Grid item xs={12} sm={12} md={6} key={pet.key}>
-              <MyPetCard name={pet.name} breed={pet.breed} type={pet.species} age={pet.petAge} />
-            </Grid>
-          ))
-        ) : (
-          <h1>no pets yet :(</h1>
-        )}
-        <Grid container direction="column" alignItems="center" gap="1rem" item>
-          <Link component={RouterLink} to={paths.addPet} style={{ textDecoration: 'none', justifyContent: 'center' }}>
-            <CustomButton text="Add pet" />
-          </Link>
+      <Box width="70vw">
+        <Grid container spacing={4} paddingLeft="40px" paddingRight="40px" gridAutoColumns="2" margin="0">
+          {pets.length > 0 ? (
+            pets.map((pet) => {
+              return (
+                <Grid item xs={12} sm={12} md={6} key={pet.key}>
+                  <MyPetCard
+                    name={pet.name}
+                    breed={pet.breed}
+                    type={pet.species}
+                    age={new Date(pet.petAge.seconds * 1000 + pet.petAge.nanoseconds / 1000000).toLocaleDateString(
+                      undefined,
+                      options,
+                    )}
+                  />
+                </Grid>
+              );
+            })
+          ) : (
+            <h1>no pets yet :(</h1>
+          )}
+          <Grid container direction="column" alignItems="center" gap="1rem" item>
+            <Link component={RouterLink} to={paths.addPet} style={{ textDecoration: 'none', justifyContent: 'center' }}>
+              <CustomButton text="Add pet" />
+            </Link>
+          </Grid>
         </Grid>
-      </Grid></Box>
+      </Box>
```

Comment:
![image](https://user-images.githubusercontent.com/95089940/151624999-2131874a-257f-453b-8c20-d4876035a234.png)

Cards with my pets must be displayed on the right side of the sideBar